src/Workspaces/MSBuildTest/MSBuildWorkspaceTests.cs

```diff
@@ -1762,10 +1762,10 @@ public void TestOpenProject_WithUnrecognizedProjectReferenceFileExtension_SkipFa
         }
 
         [ConditionalFact(typeof(VisualStudioMSBuildInstalled)), Trait(Traits.Feature, Traits.Features.MSBuildWorkspace)]
-        public async Task TestParseOptions_CSharp_LanguageVersion_Latest()
+        public async Task TestParseOptions_CSharp_LanguageVersion_Default()
         {
             CreateCSharpFiles();
-            await AssertCSParseOptionsAsync(CS.LanguageVersion.CSharp7, options => options.LanguageVersion);
+            await AssertCSParseOptionsAsync(CS.LanguageVersion.CSharp8, options => options.LanguageVersion);
```

Comment:
Can we write this more robustly so we _don't_ have to change this each and every time? (@DustinCampbell might have an opinion here.)

Reply:
I'll punt on that til next time if that's ok. We don't update major versions often.